nuxeo-drive-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-drive/nuxeo-drive-edit-button.html

```diff
@@ -66,7 +66,8 @@ <h1>[[i18n('driveEditButton.dialog.heading')]]</h1>
       },
 
       _isAvailable: function(doc, blob) {
-        return doc && blob && this.hasPermission(doc, 'Write');
+        return doc && blob && this.hasPermission(doc, 'Write')
+            && (blob.appLinks && Array.isArray(blob.appLinks) && blob.appLinks.length === 0);
```

Comment:
maybe `&& (!blob.appLinks || blob.appLinks.length === 0)`?
- there might be some cases where the `appLinks` enricher isn't sent
- if it is sent, I believe that we can safely assume that it's an array

Reply:
Thanks! Seems sensible.